features/task_validate_image.feature

```diff
@@ -15,8 +16,8 @@ Feature: Verify Enterprise Contract Tekton Tasks
         "sources": [
           {
             "policy": [
-              "github.com/enterprise-contract/ec-policies//policy/release",
-              "github.com/enterprise-contract/ec-policies//policy/lib"
+              "github.com/enterprise-contract/ec-policies//policy/release?ref=0de5461c14413484575e63e96ddb514d8ab954b5",
```

Comment:
I need to explicitly add a commit id here because report-json.json added the latest commit id if not specified.

Reply:
@simonbaird could you PTAL if this make sense to you?

Reply:
Yeah, it makes sense.